js/plates-model/model.js

```diff
@@ -102,6 +102,13 @@ export default class Model {
     this.setPlatesProp('angularVelocity', map)
   }
 
+  setDensities (densities) {
+    this.forEachPlate(plate => {
+      plate.setDensity(densities[plate.id])
+    })
+    this.plates.sort(sortByDensityDesc)
```

Comment:
I think this is why cross section might be broken (and some other things in the model).
Now, when there's swapped order of densities (hight numbers go to the bottom), I guess we should also swap order of plates array and sort them using `sortByDensityAsc`. There's an assumption that the first plate in this array is the top-most one. It's stated in the constructor, but we might add more comments.